stdlog/stdlog-intern.h

```diff
@@ -28,6 +28,14 @@
 #include <sys/un.h>
 #include "stdlog.h"
 
+#ifndef O_CLOEXEC
+#define O_CLOEXEC 0
+#endif
+
+#ifndef SOCK_CLOEXEC
+#define SOCK_CLOEXEC 0
+#endif
+
 #define __STDLOG_MSGBUF_SIZE 4096
 #ifndef STDLOG_INTERN_H_INCLUDED
```

Comment:
I know this isn't a part of your change, but I find it odd that the include guard is not at the top of the file protecting all the includes and definitions in this file. Can anyone say why it is done this way?